fre/pp/frepp.py

```diff
@@ -1,10 +1,10 @@
 ''' fre pp '''
 
 import click
-from .checkoutScript import checkoutTemplate
-from .configure_script_yaml import yamlInfo
+from .checkoutScript import _checkoutTemplate
+from .configure_script_yaml import _yamlInfo
 from .configure_script_xml import convert
-from .validate import validate_subtool
+from .validate import _validate_subtool
 from .install import install_subtool
```

Comment:
Do the click entry points for `install_subtool`, `pp_run_subtool`, and `status_subtool` need to be fixed too?

Reply:
yep, and they should follow the same clean-convention you followed for `fre make` as well! will address